aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/transferutility/TransferUtility.java

```diff
@@ -385,15 +385,16 @@ public TransferObserver download(String key, File file) {
      * @param key The key under which the object to download is stored.
      * @param file The file to download the object's data to.
      * @param listener a listener to attach to transfer observer.
+     * @param connectionType The connection type to use for the transfer.
      * @return A TransferObserver used to track download progress and state
      */
     public TransferObserver download(String bucket, String key, File file,
-            TransferListener listener) {
+            TransferListener listener, TransferConnectionType connectionType) {
```

Comment:
This would be a breaking API change for developers. Instead can you add this option through the TransferUtilityOptions object? See https://github.com/aws-amplify/aws-sdk-android/blob/master/aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/transferutility/TransferUtilityOptions.java. You can add a new data member with getter and setter and a default value.